hugegraph-rocksdb/src/main/java/org/apache/hugegraph/backend/store/rocksdb/RocksDBTables.java

```diff
@@ -218,6 +218,7 @@ public IndexTable(String database, String table) {
         }
 
         @Override
+        // TODO: why this method is same as super.eliminate() in RocksDBTable, del it?
```

Comment:
added the assert?

Reply:
> added the assert?

seems totally same in parent class?

https://github.com/apache/incubator-hugegraph/blob/e7c7558acdc8a27167dc5f2f36168a3ef4423b53/hugegraph-rocksdb/src/main/java/org/apache/hugegraph/backend/store/rocksdb/RocksDBTable.java#L114

Reply:
I checked and find out that **delete() method is override by IndexTable, we want to call super.delete() instead of this.delete() here**. we can add a NOTE here.